mmgen/models/architectures/sngan_proj/generator_discriminator.py

```diff
@@ -657,17 +687,35 @@ def init_weights(self, pretrained=None, strict=True):
                                                       map_location)
             self.load_state_dict(state_dict, strict=strict)
         elif pretrained is None:
-            if self.init_type.upper() == 'BIGGAN':
+            if self.init_type.upper() == 'STUDIO':
+                # initialization method from Pytorch-StudioGAN
```

Comment:
I suggest you give formal references with links since you provide several init types.

Reply:
Links and comments are contained in the docstring.